src/interfaces/name-check-interfaces/restricted-response-interface.ts

```diff
@@ -0,0 +1,15 @@
+export interface RestrictedResponseIF {
+  restricted_words_conditions: Array<{
+    cnd_info: Array<{
+      allow_use: string,
+      consent_rrequired: string,
```

Comment:
Typo?
Boolean?

Reply:
ah yes typo, but this value is never actually used by the UI